packages/core/src/utils/input.ts

```diff
@@ -0,0 +1,16 @@
+/** Changes a given `input`'s `value`, triggering its `onChange` */
+export const changeInputValue = (
```

Comment:
If this util is for the input, shouldn't it stay closer to the Input component? Can we move this to `Input/utils.ts`?
`HvNumberInput` is already importing other stuff from `Input`!